app/components/provider_interface/user_permission_summary_component.html.erb

```diff
@@ -5,15 +5,15 @@
         <%= description_for(permission) %>
       </dt>
       <dd class="govuk-summary-list__value">
-        <%= can_perform_permission_y_n?(permission) %>
+        <p class="govuk-body"><%= can_perform_permission_y_n?(permission) %></p>
```

Comment:
We've added this to match up with what the prototype and what the summary list component would render